src/Functions.php

```diff
@@ -105,16 +107,13 @@ function test(string $description = null, Closure $closure = null)
      * is the test description; the second argument is
      * a closure that contains the test expectations.
      *
-     * @return TestCall|TestCase|mixed
+     * @return TestCall
```

Comment:
You sure this change does not cause auto-completion issues?

Reply:
Just looking into it further. 👍🏻